### DISCUSSION TITLE
Device that is cut out from power still appears "running"

### DISCUSSION THREAD

**Discussion by Sulomus** [FlowFuse/flowfuse#1540] [Q&A] [answered]
Hello,
I have played today with FlowForge and FlowForge Device Agent. I got everything working (even the service) and I totally love it!

When I turned off the device and came back from work to check FlowForge I noticed that it is still marked as "running". 
Am I misunderstanding this label or is it a bug?

![image](https://user-images.githubusercontent.com/49107150/211895875-bbef89f2-9b04-4dfb-83f9-3c3cb967be65.png)

**Comment by hardillb** [comment, marked as answer]
This is a bug, the status label should get updated if the last seen value passes a threshold.

There is an open issue to look at it https://github.com/flowforge/flowforge/issues/1116
